### PR TITLE
Adding/deleting posts

### PR DESCRIPTION
**Pull request by likash**
Changed subscribers/subscriptions counting, adding/deleting posts, changed design of posts list on the profile page

### CODE REVIEW

File: Gruvo/Gruvo/ClientApp/src/app/app.module.ts [hunk 1]
```diff
@@ -21,7 +22,8 @@ import { AuthGuard } from '@app/auth.guard';
 import { FeedComponent } from '@app/feed/feed.component';
 import { FeedService } from '@app/feed/feed.service';
 import { RecommendationComponent } from '@app/recommendation/recommendation.component';
-
+import { CreateTweetComponent } from '@app/createtweet/createtweet.component';
```
Comment (Hunroll) [inline Gruvo/Gruvo/ClientApp/src/app/app.module.ts]: createtweet directory and createtweet.component. everything is lowercase, look https://angular.io/guide/styleguide#separate-file-names-with-dots-and-dashes

File: Gruvo/Gruvo/ClientApp/src/app/createtweet/createtweet.component.ts [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+  public postTweet(event) {
+    this.createTweetService.postTweet(this.message)
+      .subscribe(
+        tweet => console.log(tweet),
+        error => console.log(error)
+    );
+    event.preventDefault();
+    this.message = '';
+  }
+  isMessageEmpty() {
```
Comment (Hunroll) [inline Gruvo/Gruvo/ClientApp/src/app/createtweet/createtweet.component.ts]: No empty line between methods:)

File: Gruvo/Gruvo/ClientApp/src/app/createtweet/createtweet.service.ts [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+import { ITweet } from "@app/tweet/tweet.model";
+
+@Injectable()
+export class CreateTweetService {
+  constructor(
+    @Inject('CREATETWEET_POST_TWEET_URL') private createtweetPostTweetApiURL: string,
+    private http: HttpClient) { }
+
+  postTweet(message: string) {   
+    const headers = new HttpHeaders().set('Content-Type', 'application/json');
```
Comment (Hunroll) [inline Gruvo/Gruvo/ClientApp/src/app/createtweet/createtweet.service.ts]: it could be class constant as it is the same and should not be initialized on every method call

File: Gruvo/Gruvo/ClientApp/src/app/profile/profile.component.ts [hunk 4]
```diff
@@ -11,30 +12,54 @@ import { ITweet } from '@app/tweet/tweet.model';
...
+  }
 
-  ngOnInit(): void {
+  loadUserData() {
     this.profileService.getUserData()
       .subscribe((user) => {
         this.user = user;
       });
+  }
+  loadUserTweets() {
```
Comment (Hunroll) [inline Gruvo/Gruvo/ClientApp/src/app/profile/profile.component.ts]: no empty lines between methods.

File: Gruvo/Gruvo/ClientApp/src/app/profile/profile.service.ts [hunk 5]
```diff
@@ -23,10 +24,14 @@ export class ProfileService {
   getUserTweets(): Observable<ITweet[]> {
     return this.http.get<ITweet[]>(this.profileTweetsApiURL);
   }
-  getSubscriptions(): Observable<IUser[]> {
-    return this.http.get<IUser[]>(this.profileSubscriptionsApiURL);
+  getSubscriptionsQuality(): Observable<number> {
```
Comment (Hunroll) [inline Gruvo/Gruvo/ClientApp/src/app/profile/profile.service.ts]: quality or quantity?) Better use word "count" for clarity) getSubscriptionsCount etc.

File: Gruvo/Gruvo/DAL/ITweetDAO.cs [hunk 6]
```diff
@@ -12,6 +12,8 @@ public interface ITweetDAO
         IEnumerable<ReadableTweet> GetUserPosts(long id);
         ReadableTweet GetPost(long id);
 
+        Int32 GetUserPostsQuality(long id);
```
Comment (Hunroll) [inline Gruvo/Gruvo/DAL/ITweetDAO.cs]: Quality

File: Gruvo/Gruvo/Startup.cs [hunk 7]
```diff
@@ -8,6 +8,7 @@
 using Microsoft.AspNetCore.SpaServices.AngularCli;
 using Microsoft.Extensions.Configuration;
 using Microsoft.Extensions.DependencyInjection;
+using System;
```
Comment (Hunroll) [inline Gruvo/Gruvo/Startup.cs]: Unused? Why was this file changed and committed at all?

Comment (likash) [reply Gruvo/Gruvo/Startup.cs]: It was needed for  spa.Options.StartupTimeout = new TimeSpan(0, 0, 80); I forgot to remove

File: Gruvo/Gruvo/ClientApp/src/app/profile/profile.service.ts (hunk 5 above)
Comment (likash) [reply Gruvo/Gruvo/ClientApp/src/app/profile/profile.service.ts]: accidentally)
